service/src/main/java/uk/gov/hmcts/reform/fpl/model/ReviewDecision.java

```diff
@@ -13,4 +14,9 @@
     private final String hearing;
     private final CMOReviewOutcome decision;
     private final String changesRequestedByJudge;
+
+    @JsonIgnore
+    public boolean hasReviewOutcomeOf(CMOReviewOutcome reviewOutcome) {
+        return reviewOutcome.equals(decision);
+    }
```

Comment:
Do we need this? and then tests?

Reply:
Perhaps not, it's more for legibility in the controller. 

`caseData.getReviewCMODecision().getDecision().equals(SEND_TO_ALL_PARTIES)`
becomes
`caseData.getReviewCMODecision().hasReviewOutcomeOf(SEND_TO_ALL_PARTIES)`

Reply:
Agreed with James, reads much nicer 👍 
And it's generally good approach to implement little methods like that.